apps/prairielearn/python/prairielearn.py

```diff
@@ -292,18 +292,18 @@ def to_json(v, *, df_encoding_version=1, np_encoding_version=1):
             }
     elif isinstance(v, sympy.Expr):
         return phs.sympy_to_json(v)
-    elif isinstance(v, sympy.Matrix) or isinstance(v, sympy.ImmutableMatrix):
+    elif isinstance(v, (sympy.Matrix, sympy.ImmutableMatrix)):
```

Comment:
this looks like a difference in behavior to me. Was the code just broken before? It looks like `isinstance` is supposed to take a `classInfo` but it was being fed a `Tuple` before. 

(https://docs.python.org/3/library/functions.html#isinstance)

Reply:
Maybe I am misunderstanding: I have changed it to be a tuple, it used to be separate. This is preferred behavior by some of the linters.

> If classinfo is a tuple of type objects (or recursively, other such tuples) or a [Union Type](https://docs.python.org/3/library/stdtypes.html#types-union) of multiple types, return True if object is an instance of any of the types.

Reply:
or sorry, I misread that. Looks good.

Reply:
This is exactly the same behaviour. From the link you posted:

> If `classinfo` is a tuple of type objects (or recursively, other such tuples) or a Union Type of multiple types, return True if object is an instance of any of the types.